src/apps/chatter/components/chatbox/chatbox.js

```diff
@@ -1,12 +1,21 @@
 import './chatbox.less'
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { Message } from '/elements/message'
 import { useRoom } from '../../hooks/use-room'
 
 export const ChatBox = ({ roomId }) => {
   const [textInput, setTextInput] = useState('')
   const [messages, sendNewMessage] = useRoom(roomId)
 
+  useEffect(() => {
+    document.querySelector('#chatbox__message-input').focus()
+    document.querySelector('#chatbox__message-input').onkeyup = function (e) {
+      if (e.keyCode === 13) {
+        document.querySelector('.chatbox__send-button').click()
+      }
+    }
```

Comment:
should be using `refs` to access the dom node. The first example for `useRef` is for focusing an input: https://reactjs.org/docs/hooks-reference.html#useref

React has event binding: https://reactjs.org/docs/events.html here are the keyboard events: https://reactjs.org/docs/events.html#keyboard-events

basically:

```javascript
  const handleKeyUp = ev => {
    if (ev.keyCode === 13) {
      console.log('ENTER')
    }
  }

  // ...

        <input
          type="text"
          name="input"
          onKeyUp={handleKeyUp}
        />
```

Reply:
✔️ ✔️

Reply:
This took a bit to figure out how to stop the input from losing focus but I finally got it 😃